crates/cargo-bsp/src/project_model/package_dependency.rs

```diff
@@ -1,3 +1,5 @@
+//! Stores package dependencies.
```

Comment:
Structure representing a dependency of a package

Reply:
Fields _features and _uses_default_features may become handy when implementing BuildTargetDependencyModules request. Optional dependencies of the PackageDependency are included only if specific features for PackageDepenency are set. Similarly, disabling default features impact the set of dependencies of the PackageDependency.